src/view/pages/Farmer/tabs/Info/index.tsx

```diff
@@ -56,7 +42,6 @@ class FarmerInformation extends React.Component<OwnProps, OwnState> {
       farmerNotes: 'Doctor from village A',
       selectedPaymentCycle: 'Weekly',
       selectedPaymentMethod: 'Mobile',
-      // modeHandler:
     };
 
     this.changeMode.bind(this);
```

Comment:
We decided to make binding optional, so if you prefer to use arrow functions go ahead. I don't imagine I will be writing integration tests anyways so I don't think it will make a difference in the long run, and might be more consistent for you. Just be sure you understand the implications of using an arrow function (just as you should understand the implications of `function name() {}`, `{ name() {} }` and any other function notations). There are fewer gotcha's to be aware of using arrow function, so it might well be better for you guys to use them